sparql/sparql11/protocol/manifest.ttl

```diff
@@ -1,961 +1,1430 @@
+                  ht:elementName "application/rdf+xml"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/turtle"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/n-triples"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/html"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/ld+json"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "DESCRIBE query appropriate content type (expect one of: RDF/XML, Turtle, N-Triples, RDFa, JSON-LD)" .
+
+<#query_content_type_select> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "query=SELECT%20%281%20AS%20%3Fvalue%29%20%7B%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf"
+        ];
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:headers ([
+              a ht:RequestHeader;
+              ht:fieldName "Content-Type";
+              ht:fieldValue "application/sparql-results+xml, application/sparql-results+json, text/tab-separated-values, text/csv";
+              ht:headerElements ([
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+xml"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+json"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/tab-separated-values"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/csv"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "SELECT query appropriate content type (expect one of: XML, JSON, CSV, TSV)" .
+
+<#query_dataset_default_graph> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "query=ASK%20%7B%20%3Chttp%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata1.rdf%3E%20%3Fp%20%3Fo%20%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata1.rdf"
```

Comment:
Original test encodes query as a direct-POSTed request body, not a form-url-encoded one.